### DISCUSSION TITLE
Apple Silicon

### DISCUSSION THREAD

**Discussion by suggymoto** [DanielStormApps/Fanny#82] [General]
Do you have plans to release fanny for Apple Silicon? Currently it doesn't seem to work correctly. Thanks.

**Comment by mohamedshez** [comment]
I'm looking for same M1 support answer... is that possible with new Macbooks?

**Comment by anthonyboutinov** [comment]
The app looks great! Please, add support for the new Apple computers.

**Comment by Twolane2** [comment]
Appears to be dead in the water for the latest Macs.
